tamr-config.yml

```
@@ -45,3 +45,14 @@ TAMR_STORAGE_PROVIDERS: '${tamr_external_storage_providers}'
 
 # Misc
 TAMR_BIGQUERY_ENABLED: false
+
+# Backup Config
+TAMR_FILE_BASED_HBASE_BACKUP_ENABLED: ${tamr_file_based_hbase_backup_enabled}
+TAMR_BACKUP_AWS_CLI_ENABLED: ${tamr_backup_aws_cli_enabled}
+TAMR_UNIFY_BACKUP_ES: ${tamr_unify_backup_es}
+TAMR_UNIFY_BACKUP_AWS_ROLE_BASED_ACCESS: ${tamr_unify_backup_aws_role_based_access}
+TAMR_UNIFY_BACKUP_URI: s3://${tamr_data_bucket}/tamr/backups
```

Could we make the path configurable too and set the default to `tamr/backups` instead of hardcoding it in this file?
```suggestion
TAMR_UNIFY_BACKUP_URI: s3://${tamr_data_bucket}/${backup_path}
```

---

Done!